### PR TITLE
Update rubric phrasing

### PR DESCRIPTION
**Pull request by tmikeschu**
@case-eee @laurenfazah thoughts?

### CODE REVIEW

File: module4/capstone_project_overview.md [hunk 1]
```diff
@@ -53,44 +53,46 @@ At the end of the project (week 6):
 Rubric
 ------------
 
-### 1. Project Management
+### 1. Project Planning & Management
```
Comment (case-eee) [inline module4/capstone_project_overview.md]: 👍

File: module4/capstone_project_overview.md [hunk 2]
```diff
@@ -53,44 +53,46 @@ At the end of the project (week 6):
...
 
-*   4: Feature and scope related conversations happen within the tracking tool.
-*   3: Student is using well formatted user stories and moving cards through each status in realtime
-*   2: Student has used some tracking tool as a respository of information
-*   1: Student barely uses a tracking tool, or does not use one at all
+*   4: Developer has a highly detailed iteration map, sophisticated organization in their tracker tool, and exceptional git workflow
```
Comment (case-eee) [inline module4/capstone_project_overview.md]: silly and small, but you capitalize Git in the other three items so i vote you capitalize it here too to be consistent 🎉

File: module4/capstone_project_overview.md [hunk 3]
```diff
@@ -53,44 +53,46 @@ At the end of the project (week 6):
...
+*   3: Developer uses and iteration map to plan project scope, breaks down broad features into granular tasks, and exercises good Git workflow (e.g., feature branches, descriptive commits, incremental PRs)
```
Comment (case-eee) [inline module4/capstone_project_overview.md]: i think this should be "an iteration map"

really like how descriptive this one is 👍

File: module4/capstone_project_overview.md [hunk 4]
```diff
@@ -53,44 +53,46 @@ At the end of the project (week 6):
...
+*   2: Developer has a mental vision for the project iterations, shows some use of a tracker tool, and uses Git effectively
+*   1: Developer shows little evidence of project planning and management
 
 ### 2. Completion & Pace
 
-*   4: Student is proactive in understanding scope and is able to commit to stories before starting the sprint
-*   3: Student is able to set and update expectations so that there are no surprises on the last day of the sprint
-*   2: Student does not have agreed upon stories completed at the end of the sprint, but has a plan to get them done
-*   1: Student does not have agreed upon stories completed, and has no plan to complete them
+*   4: Developer delivers stories committed to before starting the sprint
```
Comment (case-eee) [inline module4/capstone_project_overview.md]: do we want to clarify what a sprint is here? or should we do that elsewhere?

File: module4/capstone_project_overview.md [hunk 5]
```diff
@@ -53,44 +53,46 @@ At the end of the project (week 6):
...
+*   3: Developer plans stories ahead of sprint and makes some scope adjustments along the way
+*   2: Developer implements a very little of the planned sprint
```
Comment (case-eee) [inline module4/capstone_project_overview.md]: grammar thing - don't need the "a" here

File: module4/capstone_project_overview.md [hunk 6]
```diff
@@ -53,44 +53,46 @@ At the end of the project (week 6):
...
+*   1: Developer does not plan or complete sprint
 
 ### 3. Implementation Quality
 
-*   4: Project demonstrates exceptionally well tested and maintainable code.
-*   3: Project exhibits tested, maintainable, well divided code. Developers are able to speak to architecture and implementation decisions.
-*   2: Project demonstrates some gaps in code quality and/or developers cannot defend their decisions.
-*   1: Project demonstrates poor factoring and/or understanding of MVC.
+*   4: Project demonstrates exceptionally well tested (where applicable) and maintainable code
```
Comment (case-eee) [inline module4/capstone_project_overview.md]: 🎉  "where applicable"

File: module4/capstone_project_overview.md [hunk 7]
```diff
@@ -53,44 +53,46 @@ At the end of the project (week 6):
...
+*   2: Project demonstrates some gaps in code quality and/or developers cannot defend their decisions
+*   1: Project demonstrates poor factoring and/or understanding of general programming principles such as MVC and OOP
 
 ### 4. Application of Techniques
 
-*   4: Project has implemented four or more major techniques that were new to the student.
-*   3: Project has implemented two major technique that were new.
-*   2: Project has a implementation in progress of one major technique that has not been previously attempted.
-*   1: Project does not implement new techniques.
+*   4: Developer builds project in a new language(s) or framework(s)
```
Comment (case-eee) [inline module4/capstone_project_overview.md]: should we clarify "entire" project?

File: module4/capstone_project_overview.md [hunk 8]
```diff
@@ -26,19 +26,19 @@ We would like to give you a fair amount of freedom on this project. We want you
...
+  *   Since accessibility is already in the rubric, it doesn't count as a new topic, but is required.
+  *   If you use something cool for your documentation (swagger, JSON-API, etc), we will accept it.
 
 Structure of each week
 ------------
 
 Each student will submit a project proposal after the "soft launch" [here](https://github.com/turingschool/ruby-submissions/tree/master/1703-b/4module/capstone_projects/proposals) by Tuesday, September 5th at 9 AM. Technical Lead's will give feedback on the scope of the project either via Github, Slack message, or in person by the end of the day.
 
-Each student will be responsible for setting prioritizes for their project and communicating those to their Technical Lead. Throughout the week, each student should be tagging their Technical Lead in PR's if they have questions or want feedback on any code they've written. 
+Each student will be responsible for setting prioritizes for their project and communicating those to their Technical Lead. Throughout the week, each student should be tagging their Technical Lead in PR's if they have questions or want feedback on any code they've written.
```
Comment (laurenfazah) [inline module4/capstone_project_overview.md]: *priorities

File: module4/capstone_project_overview.md (hunk 2 above)
Comment (tmikeschu) [reply module4/capstone_project_overview.md]: Great catch!

File: module4/capstone_project_overview.md (hunk 6 above)
Comment (laurenfazah) [reply module4/capstone_project_overview.md]: Love this whole section

**Comment by laurenfazah** [comment]
Looks great! Also added a couple tiny comments.

File: module4/capstone_project_overview.md (hunk 4 above)
Comment (tmikeschu) [reply module4/capstone_project_overview.md]: @case-eee e I think defining it elsewhere makes sense! Is each week a sprint, so a total of two sprints?

File: module4/capstone_project_overview.md (hunk 7 above)
Comment (tmikeschu) [reply module4/capstone_project_overview.md]: I had a question on that. What if they build a Rails back-end, for example, and learn Vue.js or something for their client?

Comment (case-eee) [reply module4/capstone_project_overview.md]: great point. maybe we leave it as is? i'd consider that two/three new things haha

File: module4/capstone_project_overview.md (hunk 4 above)
Comment (case-eee) [reply module4/capstone_project_overview.md]: yup! i think that sounds right

Comment (tmikeschu) [reply module4/capstone_project_overview.md]: Cool! I'll add it to logistics.

File: module4/capstone_project_overview.md [hunk 9]
```diff
@@ -19,26 +19,28 @@ Overview
...
 ---------------------
 
 We would like to give you a fair amount of freedom on this project. We want you to bulid something that you're excited about, and that you're proud of. These are a few requirements you should consider when pitching and building your project:
 
 *   You build something usable. You should be comfortable publishing your project by the time it's due in week 6. A spike or exploration into a new concept without a clear deliverable won't be accepted.
-*   You must apply at least two new techniques. New is something from the 4th module (OOJS, Security, Environments, React), or something extra-curricular (something not taught in the Turing backend engineering program).
-	-   Since accessibility is already in the rubric, it doesn't count as a new topic, but is required.
-	-   If you use something cool for your documentation (swagger, JSON-API, etc), we will accept it.
+*   You must apply at least four new techniques, or build your project in a new language and/or framework. New is something from the 4th module (OOJS, Security, Environments, React), or something extra-curricular (something not taught in the Turing backend engineering program).
```
Comment (tmikeschu) [inline module4/capstone_project_overview.md]: @laurenfazah @case-eee I just updated this as well.

File: module4/capstone_project_overview.md [hunk 10]
```diff
@@ -19,26 +19,28 @@ Overview
 
 The Capstone Project will last two weeks and will consist of student's building meaningful, publishable work of their choosing. An instructor will be assigned to each student and act as the Technical Lead.
 
+The project consists of two "sprints": the first finishes on Friday of Week 4, the other on Monday of Week 6.
```
Comment (tmikeschu) [inline module4/capstone_project_overview.md]: @laurenfazah @case-eee added

**Pull request merged**